useless-manager/manager.py

```diff
@@ -0,0 +1,173 @@
+# this code is licensed under GNU General Public License v3 (https://github.com/JymPatel/useless-manager/blob/master/LICENSE)
+
+import sys
+import pickle
+from datetime import datetime
+from termcolor import colored
+
+
+# defining global variables (will be overwrite by main() at initial run)
+users = [] # stores identities of users
+transactions = [] # stores all transactions done
+# other variables
+data_folder = "data/"
+
+
+
+class Transaction:
+    def __init__(self, amount, user_id, description=None):
+        # add transfer amount
+        try:
+            self.amount_change = float(amount)
+        except ValueError:
+            raise ValueError("amount is not float!")
+        # add user.ID if valid user exists
+        if user_id in [user.ID for user in users]:
+            self.ID = user_id
+        else:
+            raise IndexError("Invalid User ID!")
+        # add time and description
+        self.time = datetime.now()
+        self.description = description
+
+
+class Person:
+    def __init__(self, first, last, middle, entered_ID, balance=0):
+        self.balance = balance
+        # set Person name
+        if middle == "":
+            self.name = first + " " + last
+        else:
+            self.name = first + " " + middle + " " + last
```

Comment:
```suggestion
        if middle:
            self.name = f"{first} {middle} {last}"
        else:
            self.name = f"{first} {last}"
```
As discussed in CONTRIBUTING.md, Python f-strings are shorter, easier to read, and faster to execute.